bundles/aero.minova.rcp.constant/src/aero/minova/rcp/util/DateUtil.java

```diff
@@ -120,8 +120,12 @@ public static Instant getDate(Instant today, String input, Locale locale, String
 		try {
 			if (formulars.length > 0) {
 				if (formulars[0].matches("\\d*")) {
-					// Es beginnt mit eine Tagesangabe
-					startOfToday = LocalDate.ofInstant(getNumericDate(today, formulars[pos++], datePattern), ZoneId.of("UTC")).atStartOfDay();
+					try {
+						// Es beginnt mit eine Tagesangabe
+						startOfToday = LocalDate.ofInstant(getNumericDate(today, formulars[pos++], datePattern), ZoneId.of("UTC")).atStartOfDay();
+					} catch (Exception e) {

```

Comment:
Kann man so machen, ich würde aber lieber in Zeile 134 statt nur DateTimeException auch eine NullpointerException abfange. Geschachtelte try-Blöcke sollten vermieden werden, das ist unübersichtlich.

Reply:
Done